### PR TITLE
Deploy openfe 1.1.0, update deployment apparatus for OpenFreeEnergy namespace

### PR DESCRIPTION
**Pull request by dotsdl**
- **Deploy openfe 1.1.0; update deployment action for OpenFreeEnergy namespace**
- **openforcefield -> OpenFreeEnergy where appropriate**

### CODE REVIEW

**Comment by dotsdl** [comment]
Closes #307.

**Comment by dotsdl** [comment]
@ianmkenney can I get your blessing on this one? It looks like I can't actually run my changes to the Docker build Action from this branch; they may need to be merged to actually take effect. I'm happy to deal with any problems with what we've changed here in that Action once we've merged.

**Review by ianmkenney — APPROVED** [review]
LGTM

**Pull request merged**
